src/ext/processObliv/processObliv.ml

```diff
@@ -594,6 +634,12 @@ let rec codegenUncondInstr (instr:instr) : instr = match instr with
         | BNot -> setUnop "__obliv_c__setBitwiseNot" v e loc
         | LNot -> setUnop "__obliv_c__setLogicalNot" v e loc
         end
+    | TFloat(kind, a) when hasOblivAttr a ->
+        begin match op with
+        | Neg -> setUnop "__obliv_c__setNegF" v e loc
+        | BNot -> E.s (E.error "Bitwise operation on float invalid!\n")
```

Comment:
The newline is a mistake and should be removed. The reason for the error the way it is is because we want to discourage the use of bitwise operations on floats in general, not just obliv floats. GCC does not produce an error; I believe it is just undefined behavior and may surface a warning.